src/core/components/Menu.tsx

```diff
@@ -63,21 +63,29 @@ const Item = ({
   activeClassName = MenuClasses.ActiveItem,
   onClick,
   className = MenuClasses.Item,
+  linkTo,
 }: {
   children: ReactNode;
   onClick?: ((event: { preventDefault: Function }) => void) | undefined;
   className?: string;
   activeClassName?: string;
+  linkTo?: string;
```

Comment:
I'd prefer to use the same than link: `href` WDYT ?

Reply:
Yes, sounds good.